setup.py

```diff
@@ -114,7 +114,7 @@ def read_requirements(path):
     'twine'
 ]
 
-URSULA_REQUIRES = ['prometheus_client']
+URSULA_REQUIRES = ['prometheus_client', 'sentry_sdk']
```

Comment:
@KPrasch this is what we talked about previously regarding potential extra bloat when installing `prometheus` via `pip install nucypher[ursula]` - https://github.com/nucypher/nucypher/pull/2100#discussion_r441692284

Should we move `sentry_sdk` to dev-requirements? Or have `prometheus` be its own nucypher "extra"?